crates/ethereum/node/Cargo.toml

```diff
@@ -56,6 +56,7 @@ reth-payload-primitives.workspace = true
 reth-e2e-test-utils.workspace = true
 reth-rpc-eth-api.workspace = true
 reth-tasks.workspace = true
+reth-ethereum = { workspace = true, features = ["pool"] }
```

Comment:
we can't import this here either, because this is now cyclic,

we need to import the crates we need, which is the pool crate